scripts/minikube.sh

```diff
@@ -154,6 +154,8 @@ MINIKUBE_WAIT=${MINIKUBE_WAIT:-"all"}
 CPUS=${CPUS:-"$(nproc)"}
 VM_DRIVER=${VM_DRIVER:-"virtualbox"}
 CNI=${CNI:-"bridge"}
+NUM_DISKS=${NUM_DISKS:-"3"}
+DISK_SIZE=${DISK_SIZE:-"32g"}
```

Comment:
This is not supported by all VM_DRIVERS this is supported only for kvm2 and hyperkit

Reply:
Added a check for kvm2 and hyperkit 
Also added support for DISK_CONFIG var so  people can export empty string to avoid errors on old minikube versions.